src/services/ReportService.js

```diff
@@ -8,8 +11,83 @@ export default class ReportService {
    * @returns {Promise<string>} Nome do arquivo XLSX temporário.
    */
   async generateCompleteReport(institutionID) {
-    // TODO: implementar na task B03 (https://trello.com/c/VXgx1w4r)
-    // usar funções TechnologyService.exportTechnology(...), ExecutionService.exportExecutions(...), ExecutionService.exportConsolidatedExecutions(...)
-    // usar lib exceljs para gerar o arquivo XLSX (https://www.npmjs.com/package/exceljs)
+
+    //Cria arquivo e adiciona algumas propriedades
```

Comment:
Para evitar warnings desnecessários no lint, considere incluir espaço após os `//`
```suggestion
    // Cria arquivo e adiciona algumas propriedades
```